### ISSUE TITLE
Simple game examples

### ISSUE THREAD

**Issue by Matiiss** [Matiiss/pygame_examples#15]
Adding examples for games such as pong, flappy bird, "space invaders" (simplified version) and similar. This may require restructuring the project a bit since these examples shouldn't be mixed with other regular examples. Something like adding these to a game_examples directory.

The important thing here is that the code should be commented, like actually writing comments, explaining what stuff does. Not the very obvious stuff, but sometimes even that may need explanation. Why? What? How? are likely to be answered in comments. It could be decided to write these explanations in READMEs... Also maybe it would be better for something like this to have it's own repository or each game have it's own repository, but the examples of those games really shouldn't be too long, so it should be fine.

**Comment by blankRiot96** [comment]
I don't think this belongs here tbh, these are supposed to be pygame examples not full blown applications. But I can see your point, but I do think "projects" like these should have their own repositories

**Comment by Matiiss** [comment]
Those wouldn't be full blown applications, they probably don't even need menua or any of that. In case of pong for example, I feel like the code for that would be relatively small, like you just need two paddles, a ball and probably a score counter. However I don't feel like it's that kind of an example, it's a bit different so can't be added to the regular examples directory.

**Comment by blankRiot96** [comment]
Alright, maybe separate issues for each of these? It might be hard for someone to want to do this if the description is too broad
